security-considerations.md

```diff
@@ -196,13 +197,13 @@ points: auctioneer claiming [11] and unmatched orders [23]. Note that no order
 is partially matched since `clearingOrder` is not an existing order
 (c.f. [Price considerations](##price-considerations).
 
-```
+```bin
 [5]: out_settle = sellAmount * priceDenominator / priceNumerator
                  = fullAuctionedAmount * currentBidSum / fullAuctionedAmount
                  = currentBidSum
 ```
 
-We assume that the function `settleAuction` is built so that `currentBidSum` is
+The function `settleAuction` is built so that `currentBidSum` is
```

Comment:
The reason for the "assumption" is that I don't want to show that the contract is secure, but I assume it is in this context. I agree it's not very nice, but a formal proof would have to first prove that this statement is correct, and this is not trivial.